### PR TITLE
bug 1786274 - adding context to updates with RHEL

### PR DESCRIPTION
**Pull request by kalexand-rh**
https://bugzilla.redhat.com/show_bug.cgi?id=1786274

@sdodson, @gpei, will you PTAL?

### CODE REVIEW

File: modules/rhel-compute-updating.adoc [hunk 1]
```diff
@@ -6,7 +6,7 @@
 = Updating RHEL compute machines in your cluster
 
 After you update your cluster, you must update the Red Hat Enterprise Linux (RHEL)
-compute machines in your cluster.
+compute machines in your cluster. When you run the upgrade playbooks, the kubelet is updated on the specified RHEL workers and the machines reboot. This sequence ensures that the API is upgraded on the RHEL machines before the kubelet is upgraded.
```
Comment (sdodson) [inline modules/rhel-compute-updating.adoc]: I'm not sure this additional detail is necessary.

**Review by sdodson — APPROVED** [review]
/lgtm

File: modules/update-upgrading-web.adoc [hunk 2]
```diff
@@ -68,4 +72,10 @@ ifdef::between[]
 --
 +
 You might need to perform several intermediate updates until you reach the version that you want.
-endif::between[]
+ifdef::rhel[]
++
+[NOTE]
+====
+When you update a cluster that contains Red Hat Enterprise Linux worker machines, those workers temporarily become unavailable during the update process. You must run the update playbooks on those machines for the cluster to finish updating.
```
Comment (gpei) [inline modules/update-upgrading-web.adoc]: Maybe the approach 1) in https://bugzilla.redhat.com/show_bug.cgi?id=1786274#c28 is more accurate about this:  `You must run the upgrade playbook against each RHEL machine as it goes NotReady for the cluster to finish updating`.

Comment (sdodson) [reply modules/update-upgrading-web.adoc]: Lets go ahead with that, in parallel we'll work on removing this requirement. your suggested wording is fine with me.

**Comment by kalexand-rh** [comment]
@gpei, will you please take another look?

**Comment by gpei** [comment]
@kalexand-rh lgtm now, thanks for the update.

**Comment by kalexand-rh** [comment]
@gpei, thank you for the feedback!

@openshift/team-documentation, PTAL?

**Comment by ahardin-rh** [comment]
LGTM!

**Pull request merged**